.github/workflows/acs_manual_update.yml

```diff
@@ -0,0 +1,52 @@
+name: Build - ACS Manual Update
+
+on:
+  workflow_dispatch:
+    inputs:
+      data_year:
+        description: 'Release year of ACS data. e.g. For 2016-2020 acs5 data, type "2020"'
```

Comment:
Do these years have to reflect the most recent year (e.g. 2017 -2021)? Or should that just be done in another PR?

Reply:
this is just a description of the input so has no impact on the values that can be used

Reply:
Ahh I read this wrong I think, just the description. can disregard